cmd/root/root.go

```diff
@@ -124,6 +125,9 @@ Stack Trace:
 %s`, version, r, string(trace))
 	}()
 
+	// Set a cmd-exec-id value in the context
```

Comment:
That it is used as `cmd-exec-id` in the UA is not important here.

I suggest also not abbreviating here.